core/crypto.mk

```diff
@@ -48,6 +51,26 @@ CFG_CRYPTO_SHA256:=y
 endif
 endif
 
+ifeq ($(CFG_CRYPTO_CTR_DRBG),n)
```

Comment:
This is confusing. To start with, "DRBG" is in this context the same thing as "PRNG", could we stick with one name?
If it's an internal config thing for mbedTLS "DRBG" is OK (it would be nice with a "MBEDTLS" prefix then) since that's what it's called inside mbedTLS, but when it's used outside it should be "PRNG" or "RNG".

When using LTC only, `CFG_CRYPTO_CTR_DRBG` and `CFG_CRYPTO_HMAC_DRBG` aren't applicable.

Why do we have a choice between two RNGs? Wouldn't it be enough to always use the default CTR based implementation? Then we wouldn't need to have these flags at all.

Reply:
I'm even tempted to say that we should call it `CSPRNG`, since a `PRNG` lacks the unpredictable property necessary for crypto.

Reply:
Actually, those two config are only used for mbedTLS. I think I can move them to mbedtls folder, it will be some confuse in here. Do you have a different opinion?
And Of course, we can choose only one as default. But I think use two configure will be more clear for users.
For name, I am prefer to change to "PRNG" which is same as optee_os.

Reply:
Sounds good to move them into the mbedtls folder.